tests/unit/test_arithmetic_expression.cpp

```diff
@@ -889,3 +889,85 @@ TEST_F(ArithmeticTest, ListTest)
 
     ASSERT_TRUE(SIValue_IsNull(nullVal));
 }
+
+TEST_F(ArithmeticTest, ListSliceTest)
+{
+    SIValue result;
+    const char *query;
+    AR_ExpNode *arExp;
+    Record r = Record_New(0);
+
+    query = "RETURN [0,1,2,3,4,5,6,7,8,9,10][3]";
+    arExp = _exp_from_query(query);
+    result = AR_EXP_Evaluate(arExp, r);
+
+    ASSERT_EQ(T_INT64, result.type);
+    ASSERT_EQ(3, result.longval);
+
+    query = "RETURN [0,1,2,3,4,5,6,7,8,9,10][-3]";
+    arExp = _exp_from_query(query);
+    result = AR_EXP_Evaluate(arExp, r);
+
+    ASSERT_EQ(T_INT64, result.type);
+    ASSERT_EQ(8, result.longval);
+
+    query = "RETURN [0,1,2,3,4,5,6,7,8,9,10][0..3]";
```

Comment:
Add a short comment explaining what value is expected, do the same for every test query

Reply:
done